backend/src/appointment/database/models.py

```diff
@@ -203,7 +204,7 @@ class Appointment(Base):
     __tablename__ = 'appointments'
 
     id = Column(Integer, primary_key=True, index=True)
-    uuid = Column(UUIDType(native=False), default=uuid.uuid4, index=True, unique=True)
+    uuid = Column(UUID(as_uuid=True), default=uuid.uuid4, index=True, unique=True)
```

Comment:
Does this need any migration?

Reply:
No schema migration is needed. The underlying database column type (Binary(16)) remains the same, It's just that `sqlalchemy_utils.UUIDType` is changed to `sqlalchemy.types.UUID`

Reply:
I see. I think the requirement in #487 was to migrate this to 32 character hexadecimal strings using [sqlalchemy.Uuid](https://docs.sqlalchemy.org/en/20/core/type_basics.html#sqlalchemy.types.Uuid)

CC @MelissaAutumn

Reply:
Ah, I misunderstood the issue, let me migrate to `32 character hexadecimal strings` using [sqlalchemy.Uuid](https://docs.sqlalchemy.org/en/20/core/type_basics.html#sqlalchemy.types.Uuid)

cc. @devmount @MelissaAutumn

Reply:
> No schema migration is needed. The underlying database column type (Binary(16)) remains the same, It's just that `sqlalchemy_utils.UUIDType` is changed to `sqlalchemy.types.UUID`

So I was wrong here, This change will store UUIDs as 32-character hexadecimal strings in the database, and Yes, this change will require a database migration. 

Again i'm sorry for all this confusion, my bad 😓 .

cc. @devmount

Reply:
All good. Thanks anyway for your work on this!